comprehensive-getting-started/Layer_Comprehensive_Getting_Started.ipynb

```diff
@@ -0,0 +1,883 @@
+        "  final_training_df = training_df.append(external_df)\n",
+        "\n",
+        "  training_df[['target']].plot(kind='box', title='Target Quartile Analysis')\n",
+        "  layer.log({\"Box plots\": plt})\n",
+        "\n",
+        "  plt.close()\n",
+        "  \n",
+        "  return training_df "
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LnaPlmrP4Yg3"
+      },
+      "source": [
+        "Now, we are ready to take one more step to run the function above on [Layer](https://layer.ai/) Cloud. <br>\n",
+        "\n",
+        "When you run the cell below for the first time, it will create your dataset ***my_first_dataset v1.1*** and also print a link to that dataset's page on [Layer](https://layer.ai/) Web UI. Once you click on the generated link, you will see a dataset page looks like the one below:\n",
+        "\n",
+        "  <img src=\" https://github.com/layerai/examples/blob/layer-comprehensive-demo/comprehensive-getting-started/images/Screenshot%202022-07-14%20at%2012.26.03.png?raw=true\" width=\"700\" height=\"500\" />\n",
+        "\n",
+        "Once you run the cell below for the second time without changing anything in the ***create_dataset*** function's code, [Layer](https://layer.ai/) will generate a new version of the same dataset and bump up only the minor version number *(v1.1 --> v1.2)* since there will be no change in schema of the dataset. When you click on newly generated link, you will be directed to the ***my_first_dataset v1.2***'s page.\n",
+        "\n",
+        "  <img src=\" https://github.com/layerai/examples/blob/layer-comprehensive-demo/comprehensive-getting-started/images/Screenshot%202022-07-14%20at%2012.28.22.png?raw=true\" width=\"700\" height=\"500\" />"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "crHeFPUVLryD"
+      },
+      "outputs": [],
+      "source": [
+        "# --- Run your function: create_dataset remotely on Layer Infra ---\n",
+        "\n",
+        "# Run this cell twice to create my_first_dataset v1.1 & v1.2 \n",
+        "# and check them out on Layer Web UI by clicking on the links generated in the output of the cell \n",
+        "layer.run([create_dataset])"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "wjQKa3w4TK0Y"
+      },
+      "source": [
+        "Now, let's make some changes in the function body that affects the returning dataset's schema such as adding one more column. Assume that you have come up with a new feature that you think it will help improving your ML model's prediction performance. You would add this new feature into your dataset in the next iteration (or experiment) and train your model using the new version of your dataset. In the case below, we will simulate it by increasing number of features ***n_features*** in the ***make_regression*** function from 3 to 4.<br>\n",
```

Comment:
Assume that you have come up with a new feature that you think will help improve your ML model's prediction performance